packages/menu/src/SubMenu.tsx

```diff
@@ -325,9 +232,31 @@ const SubMenu: ExtendableBox<
 
     return (
       <li role="none" className={liStyle}>
-        {renderBox}
+        <MenuItem
```

Comment:
@DesignerDave Here's the issue. The root element of `MenuItem` is a `<li>`, so here we have two nested `<li>`s

Reply:
Ahh, yeah nice catch. Now the reasoning why it was the way it was makes sense. Are you gonna try to update this or revert it?

Reply:
I'm moving it to a different PR, but I'll try and fix it